apps/dataflow/src/scraper/fms/get_all_svg.py

```
@@ -0,0 +1,241 @@
+    last_exc = None
+    for attempt in range(1, retries + 1):
+        try:
+            resp = requests.get(
+                ENDPOINT,
+                params=params,
+                cookies=cookies,
+                headers=headers,
+                timeout=timeout,
+                verify=verify_ssl,
+            )
+            if resp.status_code == 200 and resp.text.strip():
```

Ruff (0.14.14)

[warning] Magic value used in comparison, consider replacing `200` with a constant variable

(PLR2004)